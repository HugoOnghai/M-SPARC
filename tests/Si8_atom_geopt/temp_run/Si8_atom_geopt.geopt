:RELAXSTEP: 1
:R(Bohr):
  1.0000000000E-01   2.1000000000E-01   1.7000000000E-01
  2.5500000000E+00   2.5500000000E+00   2.5500000000E+00
  0.0000000000E+00   5.1000000000E+00   5.1000000000E+00
  2.5500000000E+00   7.6500000000E+00   7.6500000000E+00
  5.1000000000E+00   0.0000000000E+00   5.1000000000E+00
  7.6500000000E+00   2.5500000000E+00   7.6500000000E+00
  5.1000000000E+00   5.1000000000E+00   0.0000000000E+00
  7.6500000000E+00   7.6500000000E+00   2.5500000000E+00
:F(Ha/Bohr):
 -1.6005609847E-02  -1.9614726782E-02  -1.7704897508E-02
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -9.7811150511E+00  -1.7122779927E+00  -1.9766152494E+00 
 -1.7122779927E+00  -9.9116393174E+00  -1.3618435321E+00 
 -1.9766152494E+00  -1.3618435321E+00  -9.8624283745E+00 
:RELAXSTEP: 2
:R(Bohr):
  9.7409512723E-02   2.0682538181E-01   1.6713448521E-01
  2.5500000000E+00   2.5500000000E+00   2.5500000000E+00
  0.0000000000E+00   5.1000000000E+00   5.1000000000E+00
  2.5500000000E+00   7.6500000000E+00   7.6500000000E+00
  5.1000000000E+00   0.0000000000E+00   5.1000000000E+00
  7.6500000000E+00   2.5500000000E+00   7.6500000000E+00
  5.1000000000E+00   5.1000000000E+00   0.0000000000E+00
  7.6500000000E+00   7.6500000000E+00   2.5500000000E+00
:F(Ha/Bohr):
 -1.5570623603E-02  -1.9320550184E-02  -1.7372990944E-02
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -9.7912693006E+00  -1.6701077412E+00  -1.9330857623E+00 
 -1.6701077412E+00  -9.9151058641E+00  -1.3187029187E+00 
 -1.9330857623E+00  -1.3187029187E+00  -9.8685107596E+00 
:RELAXSTEP: 3
:R(Bohr):
  3.2786268851E-02   1.2663870091E-01   9.5030820053E-02
  2.5500000000E+00   2.5500000000E+00   2.5500000000E+00
  0.0000000000E+00   5.1000000000E+00   5.1000000000E+00
  2.5500000000E+00   7.6500000000E+00   7.6500000000E+00
  5.1000000000E+00   0.0000000000E+00   5.1000000000E+00
  7.6500000000E+00   2.5500000000E+00   7.6500000000E+00
  5.1000000000E+00   5.1000000000E+00   0.0000000000E+00
  7.6500000000E+00   7.6500000000E+00   2.5500000000E+00
:F(Ha/Bohr):
 -5.5409527011E-03  -1.1889620275E-02  -9.3626700539E-03
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -9.9917173547E+00  -7.7194683693E-01  -9.9382505581E-01 
 -7.7194683693E-01  -1.0008807573E+01  -4.1104503459E-01 
 -9.9382505581E-01  -4.1104503459E-01  -1.0002735918E+01 
:RELAXSTEP: 4
:R(Bohr):
  1.0182495956E+01   1.8760912405E-02   1.0075574190E-02
  2.5500000000E+00   2.5500000000E+00   2.5500000000E+00
  0.0000000000E+00   5.1000000000E+00   5.1000000000E+00
  2.5500000000E+00   7.6500000000E+00   7.6500000000E+00
  5.1000000000E+00   0.0000000000E+00   5.1000000000E+00
  7.6500000000E+00   2.5500000000E+00   7.6500000000E+00
  5.1000000000E+00   5.1000000000E+00   0.0000000000E+00
  7.6500000000E+00   7.6500000000E+00   2.5500000000E+00
:F(Ha/Bohr):
  1.6110625684E-03  -1.7321592473E-03  -8.8008665327E-04
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.0086663501E+01  -6.7248113703E-02  -1.3055604489E-01 
 -6.7248113703E-02  -1.0086667664E+01   1.2150826417E-01 
 -1.3055604489E-01   1.2150826417E-01  -1.0086636743E+01 
:RELAXSTEP: 5
:R(Bohr):
  1.0188037001E+01   4.4956191362E-03   1.1939492400E-03
  2.5500000000E+00   2.5500000000E+00   2.5500000000E+00
  0.0000000000E+00   5.1000000000E+00   5.1000000000E+00
  2.5500000000E+00   7.6500000000E+00   7.6500000000E+00
  5.1000000000E+00   0.0000000000E+00   5.1000000000E+00
  7.6500000000E+00   2.5500000000E+00   7.6500000000E+00
  5.1000000000E+00   5.1000000000E+00   0.0000000000E+00
  7.6500000000E+00   7.6500000000E+00   2.5500000000E+00
:F(Ha/Bohr):
  1.0049004519E-03  -3.7447240035E-04  -8.7119172332E-05
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.0089085039E+01  -7.7684023736E-03  -3.1599892658E-02 
 -7.7684023736E-03  -1.0088998653E+01   8.4575590291E-02 
 -3.1599892658E-02   8.4575590291E-02  -1.0088984973E+01 
:RELAXSTEP: 6
:R(Bohr):
  1.0194886279E+01   9.1949504576E-05   1.0199235055E+01
  2.5500000000E+00   2.5500000000E+00   2.5500000000E+00
  0.0000000000E+00   5.1000000000E+00   5.1000000000E+00
  2.5500000000E+00   7.6500000000E+00   7.6500000000E+00
  5.1000000000E+00   0.0000000000E+00   5.1000000000E+00
  7.6500000000E+00   2.5500000000E+00   7.6500000000E+00
  5.1000000000E+00   5.1000000000E+00   0.0000000000E+00
  7.6500000000E+00   7.6500000000E+00   2.5500000000E+00
:F(Ha/Bohr):
  4.7122243134E-04  -9.5794380205E-06   7.0475818850E-05
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.0089493426E+01   5.3756805506E-03  -7.1745895671E-04 
  5.3756805506E-03  -1.0089492095E+01   3.5956415524E-02 
 -7.1745895671E-04   3.5956415524E-02  -1.0089492190E+01 
