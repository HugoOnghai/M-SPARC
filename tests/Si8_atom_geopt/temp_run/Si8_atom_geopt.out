***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:09:12                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.200000 10.200000 10.200000 
FD_GRID: 22 22 22
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: gaussian
SMEARING: 0.007349864
CHEB_DEGREE: 18
NSTATES: 24
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
RELAX_FLAG: 1
RELAX_METHOD: LBFGS
RELAX_NITER: 100
L_HISTORY: 20
L_FINIT_STP: 0.005000
L_MAXMOV: 0.200000
L_AUTOSCALE: 1
L_LINEOPT: 1
L_ICURV: 1.000000
TOL_RELAX: 5.00E-04
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_VELS: 1
PRINT_RESTART: 1
PRINT_RESTART_FQ: 5
PRINT_RELAXOUT: 1
OUTPUT_FILE: Si8_atom_geopt.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.463636 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8_atom_geopt.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.42 7.42 7.42
Number of atoms of type 1          :  8
Estimated total memory usage       :  37.86 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9164433547E+00        4.827E-02        4.187
2            -3.9239031286E+00        2.897E-02        0.450
3            -3.9253582481E+00        2.043E-02        0.420
4            -3.9261058959E+00        9.517E-03        0.540
5            -3.9262294892E+00        5.027E-03        0.466
6            -3.9262639786E+00        3.043E-03        0.409
7            -3.9262764421E+00        1.142E-03        0.394
8            -3.9262782721E+00        6.049E-04        0.385
9            -3.9262789841E+00        1.368E-04        0.380
10           -3.9262791183E+00        7.391E-05        0.381
11           -3.9262791570E+00        3.010E-05        0.378
12           -3.9262791636E+00        1.122E-05        0.373
13           -3.9262791695E+00        4.596E-06        0.382
14           -3.9262791737E+00        2.676E-06        0.373
15           -3.9262791783E+00        1.050E-06        0.375
16           -3.9262791725E+00        7.664E-07        0.368
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9262791725E+00 (Ha/atom)
Total free energy                  : -3.1410233380E+01 (Ha)
Band structure energy              :  1.6585687893E+00 (Ha)
Exchange correlation energy        : -9.8385132513E+00 (Ha)
Self and correction energy         : -1.0218522880E+02 (Ha)
Entropy*kb*T                       : -3.8680870422E-03 (Ha)
Fermi level                        :  2.4284376875E-01 (Ha)
Average force                      :  3.8616310695E-03 (Ha/Bohr)
Maximum force                      :  3.0893048556E-02 (Ha/Bohr)
Time for force calculation         :  0.798 (sec)
Pressure                           :  9.8517275810E+00 (GPa)
Max stress                         :  9.9116393174E+00 (GPa)
Time for stress calculation        :  1.961 (sec)
Relax time                         :  15.192 (sec)
=====================================================================
                    Self Consistent Field (SCF#2)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9262917952E+00        2.098E-03        0.687
2            -3.9262972896E+00        4.906E-04        0.405
3            -3.9262975826E+00        1.215E-04        0.397
4            -3.9262976050E+00        5.131E-05        0.394
5            -3.9262976131E+00        2.393E-05        0.414
6            -3.9262976015E+00        7.291E-06        0.389
7            -3.9262976199E+00        2.015E-06        0.380
8            -3.9262976108E+00        6.524E-07        0.381
Total number of SCF: 8     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9262976108E+00 (Ha/atom)
Total free energy                  : -3.1410380887E+01 (Ha)
Band structure energy              :  1.6597207479E+00 (Ha)
Exchange correlation energy        : -9.8390235477E+00 (Ha)
Self and correction energy         : -1.0218524431E+02 (Ha)
Entropy*kb*T                       : -3.8031096200E-03 (Ha)
Fermi level                        :  2.4288729465E-01 (Ha)
Average force                      :  3.7863834846E-03 (Ha/Bohr)
Maximum force                      :  3.0291067877E-02 (Ha/Bohr)
Time for force calculation         :  0.738 (sec)
Pressure                           :  9.8582953081E+00 (GPa)
Max stress                         :  9.9151058641E+00 (GPa)
Time for stress calculation        :  2.739 (sec)
Relax time                         :  7.461 (sec)
=====================================================================
                    Self Consistent Field (SCF#3)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9238317813E+00        4.672E-02        0.892
2            -3.9265500098E+00        7.220E-03        0.418
3            -3.9266197146E+00        2.969E-03        0.408
4            -3.9266322743E+00        1.259E-03        0.404
5            -3.9266336186E+00        5.881E-04        0.397
6            -3.9266338452E+00        1.795E-04        0.397
7            -3.9266338687E+00        8.312E-05        0.388
8            -3.9266338839E+00        2.178E-05        0.474
9            -3.9266338861E+00        7.128E-06        0.419
10           -3.9266338899E+00        2.586E-06        0.366
11           -3.9266338849E+00        1.400E-06        0.371
12           -3.9266338874E+00        4.210E-07        0.383
Total number of SCF: 12    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9266338874E+00 (Ha/atom)
Total free energy                  : -3.1413071099E+01 (Ha)
Band structure energy              :  1.6822510069E+00 (Ha)
Exchange correlation energy        : -9.8491789558E+00 (Ha)
Self and correction energy         : -1.0218588420E+02 (Ha)
Entropy*kb*T                       : -2.3793864458E-03 (Ha)
Fermi level                        :  2.4368697576E-01 (Ha)
Average force                      :  2.0144975244E-03 (Ha/Bohr)
Maximum force                      :  1.6115980195E-02 (Ha/Bohr)
Time for force calculation         :  0.417 (sec)
Pressure                           :  1.0001086948E+01 (GPa)
Max stress                         :  1.0008807573E+01 (GPa)
Time for stress calculation        :  1.616 (sec)
Relax time                         :  8.330 (sec)
=====================================================================
                    Self Consistent Field (SCF#4)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9238119919E+00        5.509E-02        0.666
2            -3.9264120955E+00        2.762E-02        0.665
3            -3.9266599456E+00        1.770E-02        0.391
4            -3.9267449783E+00        4.664E-03        0.393
5            -3.9267640826E+00        1.314E-03        0.388
6            -3.9267658751E+00        4.224E-04        0.386
7            -3.9267660724E+00        2.007E-04        0.385
8            -3.9267661082E+00        9.458E-05        0.388
9            -3.9267660993E+00        2.840E-05        0.385
10           -3.9267660875E+00        1.245E-05        0.390
11           -3.9267660814E+00        2.572E-06        0.384
12           -3.9267660951E+00        1.080E-06        0.381
13           -3.9267660901E+00        6.310E-07        0.371
Total number of SCF: 13    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9267660901E+00 (Ha/atom)
Total free energy                  : -3.1414128721E+01 (Ha)
Band structure energy              :  1.6932314222E+00 (Ha)
Exchange correlation energy        : -9.8543159918E+00 (Ha)
Self and correction energy         : -1.0218678426E+02 (Ha)
Entropy*kb*T                       : -1.5241649067E-03 (Ha)
Fermi level                        :  2.4387330033E-01 (Ha)
Average force                      :  3.1549689911E-04 (Ha/Bohr)
Maximum force                      :  2.5239751929E-03 (Ha/Bohr)
Time for force calculation         :  0.415 (sec)
Pressure                           :  1.0086655969E+01 (GPa)
Max stress                         :  1.0086667664E+01 (GPa)
Time for stress calculation        :  1.605 (sec)
Relax time                         :  8.051 (sec)
=====================================================================
                    Self Consistent Field (SCF#5)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9267675505E+00        1.009E-03        1.369
2            -3.9267685074E+00        5.104E-04        0.929
3            -3.9267685804E+00        3.442E-04        0.798
4            -3.9267686040E+00        8.795E-05        0.444
5            -3.9267686142E+00        2.899E-05        0.385
6            -3.9267686152E+00        8.679E-06        0.373
7            -3.9267686175E+00        4.128E-06        0.360
8            -3.9267686079E+00        1.707E-06        0.359
9            -3.9267686149E+00        5.236E-07        0.359
Total number of SCF: 9     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9267686149E+00 (Ha/atom)
Total free energy                  : -3.1414148919E+01 (Ha)
Band structure energy              :  1.6934950339E+00 (Ha)
Exchange correlation energy        : -9.8544432253E+00 (Ha)
Self and correction energy         : -1.0218681742E+02 (Ha)
Entropy*kb*T                       : -1.5014582080E-03 (Ha)
Fermi level                        :  2.4387174239E-01 (Ha)
Average force                      :  1.3449234685E-04 (Ha/Bohr)
Maximum force                      :  1.0759387748E-03 (Ha/Bohr)
Time for force calculation         :  0.416 (sec)
Pressure                           :  1.0089022888E+01 (GPa)
Max stress                         :  1.0089085039E+01 (GPa)
Time for stress calculation        :  1.623 (sec)
Relax time                         :  8.058 (sec)
=====================================================================
                    Self Consistent Field (SCF#6)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9267690040E+00        2.935E-04        0.444
2            -3.9267690451E+00        1.983E-04        0.434
3            -3.9267690538E+00        1.340E-04        0.386
4            -3.9267690519E+00        3.517E-05        0.384
5            -3.9267690545E+00        1.542E-05        0.369
6            -3.9267690243E+00        6.060E-06        0.369
7            -3.9267690505E+00        1.771E-06        0.363
8            -3.9267690528E+00        4.364E-07        0.362
Total number of SCF: 8     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9267690528E+00 (Ha/atom)
Total free energy                  : -3.1414152422E+01 (Ha)
Band structure energy              :  1.6935572141E+00 (Ha)
Exchange correlation energy        : -9.8544702818E+00 (Ha)
Self and correction energy         : -1.0218687035E+02 (Ha)
Entropy*kb*T                       : -1.4963476896E-03 (Ha)
Fermi level                        :  2.4387160140E-01 (Ha)
Average force                      :  5.9569967590E-05 (Ha/Bohr)
Maximum force                      :  4.7655974072E-04 (Ha/Bohr)
Time for force calculation         :  0.591 (sec)
Pressure                           :  1.0089492570E+01 (GPa)
Max stress                         :  1.0089493426E+01 (GPa)
Time for stress calculation        :  1.625 (sec)
Relax time                         :  5.681 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  54.070 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
